tests/output/ConsoleInteractionTest.py

```diff
@@ -174,6 +175,44 @@ def test_require_settings(self):
 
             self.assertEqual(generator.last_input, 2)
 
+    def test_print_bears(self):
```

Comment:
Although this is a good way to test your changes, it is not the best.

You should focus on what your change is, that is printing the bears in sorted order, this means to have testcases with both uppercase and lowercase (ideally 3-5) and just print the bear names.

Also please test the `show_bears` function.